features/facebookImport/rollup.config.js

```diff
@@ -71,7 +71,7 @@ export default (commandLineArgs) => {
                     JSON.stringify(
                         process.env.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION ||
                             fallbackAuthorization
-                    ),
+                    ) | "",
```

Comment:
if there's no secret key for "auth", we set it to an empty string, so it's not `null` locally.